project/scripts/genDocs

```diff
@@ -57,10 +57,10 @@ if [ "$1" = "test" ]; then
 
     # add all contents of $PWD to commit
     git add -A
-    git commit -m "Update gh-pages site for $GIT_HEAD"
+    git commit -m "Update gh-pages site for $GIT_HEAD" || true
```

Comment:
instead of allowing it to silently fail, it would be nice to make it a separate task